ruff_sync.py

```diff
@@ -94,6 +94,26 @@ def _get_cli_parser() -> ArgumentParser:
     return parser
 
 
+def github_url_to_raw_url(url: URL) -> URL:
+    """
+    Convert a GitHub URL to its corresponding raw content URL
+
+    Args:
+        url (URL): The GitHub URL to be converted.
+
+    Returns:
+        URL: The corresponding raw content URL.
+    """
+    url_str = str(url)
+    if "github.com" in url_str and "/blob/" in url_str:
+        raw_url_str = url_str.replace("github.com", "raw.githubusercontent.com").replace(
+            "/blob/", "/"
+        )
+        return httpx.URL(raw_url_str)
+    else:
+        return url
+
+
 async def download(url: URL, client: httpx.AsyncClient) -> StringIO:
```

Comment:
<a href="https://sourcery.ai"><img src="https://sourcery.ai/favicon.svg" height="15" alt="Sourcery logo" /></a>  **suggestion (docstrings):** Please update the docstring for function: `download`

Reason for update: The function now indirectly supports downloading raw content from GitHub URLs due to the new github_url_to_raw_url function.

Suggested new docstring:
```
"""Download a file from a URL and return a StringIO object. Now supports GitHub URLs by converting them to their raw content equivalents."""
```